<!-- fake imu node launch file -->

<launch>

  <node pkg="nostop_kinect_sensor" type="fake_imu_node" name="fake_imu_red_blue" required="true" output="screen" >
    <param name="robot_name" value="red_blue"/>
    <param name="frequency" value="20"/><!--if is bigger than the frequencies of imu or camera, it doesn't work-->
  </node>
  
</launch>